### DISCUSSION TITLE
Confused about config.xml hierarchy

### DISCUSSION THREAD

**Discussion by Libarch** [gerbera/gerbera#1770] [Q&A] [answered]
Hello, I have just started finding my way around this impressive software (v1.8.2 on Opensuse Tumbleweed). The documentation is very comprehensive and mostly quite clear, but I am confused about the hierarchy of xml elements. Is there a template or something similar somewhere which clarifies which elements are child elements of which? For example, on the 'Server Options' page of the docs are all elements (apart from `<server>` itself of course) children of `<server>`?

**Comment by KarlStraussberger** [comment, marked as answer]
The easiest way is to run `gerbera --create-config` which prints out the core configuration.

If you really want to learn all the details go through https://raw.githubusercontent.com/gerbera/gerbera/master/config/config2.xsd
